app/src/userpages/components/StreamPage/Edit/index.jsx

```diff
@@ -74,6 +74,19 @@ const PreviewDescription = styled(Translate)`
     max-width: 660px;
 `
 
+const mapStatus = (state) => {
```

Comment:
`mapStatus` is defined 3 times in this PR. Any chance for a shared utility for it? `getStreamActivityStatus` or `mapStateToStatusIconStatus` – 😂.

Reply:
Good point - I changed the stream edit view to use the same helper from list view, the others are slightly different so left them separate.